README.rst

```diff
@@ -44,6 +44,57 @@ Server installed with PostgreSQL database
           user: grafana
           password: passwd
 
+Server installed with LDAP authentication and all authenticate users are
+administrators
+
+.. code-block:: yaml
+
+    grafana:
+      server:
+        enabled: true
+        admin:
+          user: admin
+          password: passwd
+        auth: ldap
+        ldap:
+          host: '127.0.0.1'
+          port: 389
+          use_ssl: false
+          bind_dn: "cn=admin,dc=grafana,dc=org"
+          bind_password: "grafana"
+          user_search_filter: "(cn=%s)"
+          user_search_base_dns:
+          - "dc=grafana,dc=org"
+
+Server installed with LDAP authentication and support for authorization
```

Comment:
s/LDAP authentication and support for authorization/LDAP for authentication and authorization/